portfolio/src/main/webapp/task.js

```diff
@@ -63,16 +63,23 @@ function createNewTask() {
   }
 
   const newTask = new Task(
-      name,
-      description,
-      getDurationMinutes(length, lengthUnit),
-      priority);
-  
+      name, description, getDurationMinutes(length, lengthUnit), priority);
+
   updateTaskList(newTask, lengthUnit);
 }
 
+/** Shows the "Task Added" header. */
+function showTaskAddedHeader() {
+  const $taskListHeader = $('#task-added-header');
```

Comment:
We can merge these lines into

$('#task-added-header').removeClass('d-none');

Reply:
Done.